methplotlib/import_methylation.py

```diff
@@ -24,41 +25,44 @@ def read_meth(filename, name, window, smoothen=5):
     """
     try:
         table = pd.read_csv(filename, sep="\t")
+        gr = pr.PyRanges(table.rename(columns={"start": "Start", "chromosome": "Chromosome", "end": "End", "Strand": "strand"}))
         logging.info("Read the file in a dataframe.")
-        if window:
```

Comment:
there used to be `if window:` here, since window can be None when no slicing has to be done... when this function is used for e.g. QC purposes

Reply:
Just using methplotlib CLI would never have window=None in its current implementation, but it makes sense to have the option when reading methylation files

Reply:
Okay, I have added the `if window` back :)